src/scenes/char_selection.js

```diff
@@ -0,0 +1,239 @@
+//Import statements
+import lance1Portrait from "../assets/lance1Portrait.png";
+import lance2Portrait from "../assets/lance2Portrait.png";
+import sword1Portrait from "../assets/sword1Portrait.png";
+import sword2Portrait from "../assets/sword2Portrait.png";
+import axe1Portrait from "../assets/axe1Portrait.png";
+import axe2Portrait from "../assets/axe2Portrait.png";
+import startButton from "../assets/startButton.png";
+import bg from "../assets/selectionBG.png";
+import lanceImg1 from "../assets/lance1.png";
+import lanceImg2 from "../assets/lance2.png";
+import swordImg1 from "../assets/sword2.png";
+import swordImg2 from "../assets/sword1.png";
+import axeImg1 from "../assets/axe1.png";
+import axeImg2 from "../assets/axe2.png";
+
+//Custom Button class for Phaser 3
+class Button extends Phaser.GameObjects.Image { 
+	//When creating a new custom button, follow the format below:
+		//let variableName = new Button(this, X position, Y position, Width of button, Height of button, Custom button image, this.buttonCallback);
+
+	//Constructor for the button
+    constructor(context, xPosition, yPosition, width, height, image, callback) {
+        super(context, xPosition, yPosition, image);
+        //Set width and height
+       	this.displayWidth = width;
+        this.displayHeight = height;
+
+        //Set callback and scope
+        this.myCallback = callback;
+        this.myScope = context; // scope
+ 
+ 		//Create event listeners for the button
+        this.setInteractive();
+
+        //When the mouse is hovered over, enlarge the image by 10x10 pixels
+        this.on('pointerover', function (event) {
+			this.displayWidth = width + 10;
+            this.displayHeight = height + 10;
+			
+		});
+
+        //When the mouse is no longer hovering over the image, reset it to the default width and height
+		this.on('pointerout', function (event) {
+			this.displayWidth = width;
+            this.displayHeight = height;
+			
+		});
+
+		//Add the button to the scene
+        context.add.existing(this);
+    }
+
+}
```

Comment:
A custom button, I haven't tested to see if you can use this class outside of this scene. If we can't it can be moved to a separate file later.